app/src/main/java/com/codecademy/movieapp/models/IndividualMovie.kt

```diff
@@ -0,0 +1,14 @@
+package com.codecademy.movieapp.models
+
+import com.google.gson.annotations.SerializedName
+
+data class IndividualMovie(
+    val id: Int,
+    val title :String,
+    val overview : String,
+    @SerializedName("poster_path")
+    val poster : String,
+    val release_date: String,
+    @SerializedName("vote_average")val rating : Int)
```

Comment:
```suggestion
    @SerializedName("vote_average")val rating : Float)
```
It should be Float since it could be 6.9 with a decimal point